src/cljs/rems/spa.cljs

```diff
@@ -336,7 +334,6 @@
     (lazy-load-data!)
     [:div
      [nav/navigation-widget]
-     [logo]
```

Comment:
This breaks backwards compatibility so we should keep it until it is configurable.

Reply:
may i ask how? this is not obvious to me since i am just putting the logo component into another

Reply:
This removes the hero logo from the main content. Then it is not the same as before for those installations that do not want this logo in the navbar. The default should be that if someone doesn't do the config change for the new feature then nothing changes. I.e. the hero logo stays where it has been.

Reply:
ok, I am confused, @opqdonut can you maybe help me?

Reply:
Looks to me like @marharyta has changed the behaviour so that the hero image shows up only in the login page. I think that's a reasonable change. However @Macroz would prefer the look of the site to _stay the same_ if the theme does not take `:upper-left-logo` into use. I think that's also reasonable and will make deploying the change easier (won't need to get an OK for the changed visuals from user organizations).

Would this work for you both?

- if `:upper-left-logo` is set
  - display hero image on login page
  - don't display hero on other pages
- if `:upper-left-logo` is not set
  - display hero image on all pages

Reply:
i am sorry, but i still do not get it, the look is the same

Reply:
this is the difference I see.

master:
![master-catalogue](https://user-images.githubusercontent.com/207385/104585283-7b4c8080-566c-11eb-91a1-7b8a9e3b92f5.png)

this PR:
![branch-catalogue](https://user-images.githubusercontent.com/207385/104585307-80a9cb00-566c-11eb-87d7-bef0ba6d8c0e.png)

Reply:
that is weird, it should show everywhere (home, fake login, every screen) or then home, fake login and navbar

Reply:
let me check

Reply:
I would rather have these two logos configurable separately. There is no need to couple them together.

Reply:
i see, I agree

Reply:
I fixed the part that where the logo should show the logo on home

Reply:
The fix has a problem in that it duplicates the logo. These two `when` statements are unrelated so the component can appear twice.
![doubled](https://user-images.githubusercontent.com/823661/105314934-43db5800-5bc7-11eb-9140-18866eb72977.png)

Reply:
I did fix that, pushing changes now